main/templates.tf

```diff
@@ -1,60 +1,75 @@
-data "template_file" "beekeeper-path-scheduler-apiary-container-definition" {
-  template = "${file("${path.module}/ecs-config/container-definition.json")}"
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+data "template_file" "beekeeper_path_scheduler_container_definition" {
+  template = "${file("${path.module}/files/container-definition.json")}"
 
   vars {
```

Comment:
@givanovexpe hey, is it okay if I change all the template file vars blocks to this:

```
vars = { ...
```

getting this error in egdl plans:

```
Error: Unsupported block type
--
on .terraform/modules/beekeeper/main/templates.tf line 30, in data "template_file" "beekeeper_path_scheduler_config":
30:   vars {

Blocks of type "vars" are not expected here. Did you mean to define argument
"vars"? If so, use the equals sign to assign it a value.
```

Reply:
looks like terraform 11 specifies no equals (https://www.terraform.io/docs/configuration-0-11/interpolation.html#templates) but terraform 12 requires it.